packages/fluentui/react-northstar/src/themes/teams/components/Carousel/carouselNavigationStyles.ts

```diff
@@ -18,7 +18,12 @@ const carouselNavigationStyles: ComponentSlotStylesPrepared<CarouselNavigationPr
       backgroundColor: v.backgroundColor || 'inherit',
       listStyleType: 'none',
       justifyContent: 'center',
-
+      ...(!vertical &&
+        thumbnails && {
+          justifyContent: 'start',
+          transform: `translateX(${pxToRem(v.width / 2 - 40 - +p.activeIndex * 75)})`,
```

Comment:
This is annoying me, feel like a little hack to center the active element. Any better ideas @pompomon @mnajdova @silviuaavram

Reply:
Please could you express those magic numbers (40, 75) through variables? Because currently I have no idea why you are using those values :)
Other than that I think the approach is OK

Reply:
40 is padding of the navigation container + half of thumbnail. 75 is thumbnail total + margin of the sibling

Reply:
Will change it

Reply:
Done